app/components/add-contributor/component.js

```diff
@@ -1,41 +1,58 @@
 import Ember from 'ember';
+import Contributor from 'kredits-web/models/contributor';
 
-export default Ember.Component.extend({
+const {
+  Component,
+  isPresent,
+  inject: {
+    service
+  },
+  computed
+} = Ember;
+
+export default Component.extend({
 
-  id: null,
-  realName: null,
-  address: null,
-  ipfsHash: null,
-  isCore: false,
+  kredits: service(),
 
+  newContributor: null,
   inProgress: false,
 
-  isValidId: function() {
-    return Ember.isPresent(this.get('id'));
-  }.property('id'),
+  isValidAddress: function() {
+    return this.get('kredits.web3')
+               .isAddress(this.get('newContributor.address'));
+  }.property('kredits.web3', 'newContributor.address'),
 
-  isValidRealName: function() {
-    return Ember.isPresent(this.get('realName'));
-  }.property('realName'),
+  isValidName: function() {
+    return isPresent(this.get('newContributor.name'));
+  }.property('newContributor.name'),
 
-  isValidAddress: function() {
-    return this.get('kredits.web3Instance').isAddress(this.get('address'));
-  }.property('address'),
+  isValidURL: function() {
+    return isPresent(this.get('newContributor.url'));
+  }.property('newContributor.url'),
 
```

Comment:
In theory yes (there's e.g. `"format": "uri"`), but it'd be much easier here to just have some simple regexp for HTTP URIs imo (which also keeps the build size down).